diff --git a/.vscode/settings.json b/.vscode/settings.json
new file mode 100644
index 0000000000000000000000000000000000000000..9e26dfeeb6e641a33dae4961196235bdb965b21b
--- /dev/null
+++ b/.vscode/settings.json
@@ -0,0 +1 @@
+{}
\ No newline at end of file
diff --git a/lib/fixer.js b/lib/fixer.js
index 27682e9611afd2d6b30469c3d4f4e3972f873d22..ecb8b26685b76278fd1c69c519420f8631e75187 100644
--- a/lib/fixer.js
+++ b/lib/fixer.js
@@ -1,5 +1,4 @@
 var semver = require("semver")
-var validateLicense = require('validate-npm-package-license');
 var hostedGitInfo = require("hosted-git-info")
 var isBuiltinModule = require("resolve").isCore
 var depTypes = ["dependencies","devDependencies","optionalDependencies"]
@@ -303,9 +302,6 @@ var fixer = module.exports = {
         data.license.trim() === ''
       ) {
         this.warn("invalidLicense")
-      } else {
-        if (!validateLicense(data.license).validForNewPackages)
-          this.warn("invalidLicense")
       }
     }
   }