diff --git a/lib/fixer.js b/lib/fixer.js
index 97c26b26b461dc726622f7f2ae026a1b2c4716ad..a127dc8bf60c36042ceca02ac360d8b3d5e0e450 100644
--- a/lib/fixer.js
+++ b/lib/fixer.js
@@ -1,6 +1,5 @@
 var isValidSemver = require('semver/functions/valid')
 var cleanSemver = require('semver/functions/clean')
-var validateLicense = require('validate-npm-package-license')
 var hostedGitInfo = require('hosted-git-info')
 var isBuiltinModule = require('is-core-module')
 var depTypes = ['dependencies', 'devDependencies', 'optionalDependencies']
@@ -334,9 +333,6 @@ module.exports = {
     ) {
       return this.warn('invalidLicense')
     }
-    if (!validateLicense(license).validForNewPackages) {
-      return this.warn('invalidLicense')
-    }
   },
 }
 